src/Core/src/Platform/Android/ContextExtensions.cs

```diff
@@ -124,12 +124,14 @@ static float ToPixelsUsingMetrics(double dp)
 
 		public static (int left, int top, int right, int bottom) ToPixels(this Context context, Graphics.Rect rectangle)
 		{
+			var left = (int)context.ToPixels(rectangle.Left);
```

Comment:
The device test ArrangesContentWithoutOverlapAndWithProperSize is failing:
`Assert.True() Failure\nExpected: True\nActual:   False`
<img width="361" alt="image" src="https://github.com/user-attachments/assets/de2e348a-6be5-4ac4-ace9-d8725a556f7a" />
The test use this extension method, could you review it?